mappings/18w05a#18w06a.tinydiff

```diff
@@ -12640,17 +12640,17 @@ c	net/minecraft/unmapped/C_4480345		net/minecraft/world/gen/feature/decorator/Ne
 		p	5			p_5
 		p	6			p_6
 		p	7			p_7
-c	net/minecraft/unmapped/C_4569443		net/minecraft/unmapped/C_4569443
-	f	Lnet/minecraft/unmapped/C_4569443;	f_0283218		f_0283218
-	f	Lnet/minecraft/unmapped/C_4569443;	f_2652839		f_2652839
-	f	Lnet/minecraft/unmapped/C_4569443;	f_4164293		f_4164293
-	f	Lnet/minecraft/unmapped/C_4569443;	f_4566055		f_4566055
-	f	Lnet/minecraft/unmapped/C_4569443;	f_1957439		f_1957439
-	f	Lnet/minecraft/unmapped/C_4569443;	f_4494099		f_4494099
-	f	Lnet/minecraft/unmapped/C_4569443;	f_8776067		f_8776067
-	f	Lnet/minecraft/unmapped/C_4569443;	f_0746879		f_0746879
-	f	Lnet/minecraft/unmapped/C_4569443;	f_4663157		f_4663157
-	f	Ljava/util/List;	f_0504422		f_0504422
+c	net/minecraft/unmapped/C_4569443		net/minecraft/client/recipebook/RecipeBookGroup
```

Comment:
Just a note for something to look at later: we should look at all crafting and recipe stuff to make sure it's in the places you'd expect. e.g. I placed `ClientRecipeBook` in `/client/crafting/` so either that should be moved into `/client/recipebook/` or this should be moved into `/client/crafting/` or something else.

Reply:
right, i had some merge conflicts with the changes you made some hours ago and i forgot to update the package here, crafting is more accurate since it's what we have in /net/minecraft/**crafting**